docs/whats-new.asciidoc

```diff
@@ -4,165 +4,154 @@
+* Improvements to the *Overview* tab:
+** *Reason statement shown in rendered view* - The alert rendered view displays event details, such as the alert reason statement, file paths, or process arguments, to provide context for the alert. You can take action (such as Add to Timeline) on individual fields in the statement. 
+** *Event renderer added to the Overview tab* - The event renderer displays relevant event details to provide context for the alert, such as file paths or process arguments. You can take actions on any of the fields provided.
++
+[role="screenshot"]
+image::whats-new/images/8.5/render-view.png[Alert render view]
++ 
+* If you have a http://elastic.co/pricing[Platinum subscription] or higher, these details are included in the *Insights* section:
 
-*New sub-feature privilege for cases*
+** *Alerts related by session ID* - Shows the ten most recent alerts generated during the same session. These alerts share the same session ID, a unique ID for tracking a given Linux session. To use this feature, you must enable the *Include session data* setting in your {security-guide}/install-endpoint.html#add-security-integration[{elastic-defend} integration policy]. 
 
-The *Delete cases and comments* sub-feature privilege determines whether a user can delete cases and comments. Users with current `All` access to cases are automatically granted the delete privilege upon upgrading to 8.4. However, users with current `read` access to cases are not automatically granted the delete privilege upon upgrading to 8.4. An admin can modify these user privileges.
+** *Alerts related by process ancestry* - Shows alerts that are related by process events on the same linear branch. 
 
-[role="screenshot"]
-image::whats-new/images/8.4/cases-privs.png[Cases privileges]
+** *Risk score data included* - Risk score classification data is displayed in the *Enriched data* section. If the current risk classification has changed from the original, both scores appear to show the difference. 
 
 [discrete]
-== Endpoint enhancements
+== Indicators page enabled by default 
 
-*New credential hardening protection*
-
-You can now configure {security-guide}/configure-endpoint-integration-policy.html#attack-surface-reduction[credential hardening protection] in an integration policy. Credential hardening prevents attackers from stealing credentials stored in Windows system process memory. Turn on the toggle to remove any overly permissive access rights that aren’t required for standard interaction with the Local Security Authority Subsystem Service (LSASS).
+The Indicators page, which provides a centralized view for threat intelligence analysts to view and investigate indicators of compromise (IoCs), is now enabled by default. To learn more about this feature, refer to {security-guide}/indicators-of-compromise.html[Indicators of compromise]. 
```

Comment:
"Now enabled by default" sounds like this was available in previous versions, but I thought the Indicators page is brand-new for 8.5. I can't remember whether this was in tech preview or behind feature flags for 8.4, but I don't think we documented it, so this is prob the first many readers are hearing about it.